tests/queries/0_stateless/00419_show_sql_queries.sh

```diff
@@ -8,3 +8,4 @@ $CLICKHOUSE_CLIENT -q "SHOW PROCESSLIST" &>/dev/null
 $CLICKHOUSE_CLIENT -q "SHOW DATABASES" &>/dev/null
 $CLICKHOUSE_CLIENT -q "SHOW TABLES" &>/dev/null
 $CLICKHOUSE_CLIENT -q "SHOW ENGINES" &>/dev/null
+$CLICKHOUSE_CLIENT -q "SHOW FUNCTIONS" &>/dev/null
```

Comment:
I think this test is not enough, it doesn't test the correctness of the result of the query and it doesn't test `LIKE/ILIKE` clauses. Let's also create a separate test for this